assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -629,7 +629,7 @@ che.server.secure_exposer=default
 che.server.secure_exposer.jwtproxy.token.issuer=wsmaster
 che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
-che.server.secure_exposer.jwtproxy.image=eclipse/che-jwtproxy:latest
+che.server.secure_exposer.jwtproxy.image=quay.io/eclipse/che-jwtproxy:latest
```

Comment:
are you going to use a fixed tag(sha based) or ```latest```?

Reply:
Well, it was latest before and I could pull the image using latest, so I thought that we should keep that way. If you prefer the tag, I don't see a reason not to either. In fact, I'd prefer the tag, but didn't want to change the existing practice.

Reply:
+1 to start using the particular tag instead of latest

Reply:
yes. I prefer a tag (sha). Can you change it, please?